gooddata-sdk/docs/services/catalog-data-source.rst

```diff
@@ -228,6 +228,22 @@ The *gooddata_sdk.catalog_data_source* supports the following action API calls:
 
     Invalidate cache of your computed reports to force your analytics to be recomputed.
 
+* ``scan_data_source(data_source_id: str, scan_request: CatalogScanModelRequest = CatalogScanModelRequest(), report_warnings: bool = False)``
+
+    Returns *CatalogScanResultPdm*.
+
+    Scan data source specified by its id and optionally by specified scan request. Additional parameter report_warnings can be passed to suppress or to report warnings. The parameter report_warnings is by default set to False.
+
+* ``scan_and_put_pdm(data_source_id: str, scan_request: CatalogScanModelRequest = CatalogScanModelRequest())``
+
+    This method combines *scan_data_source* and *put_declarative_pdm* methods.
+
+* ``scan_schemata(data_source_id: str)``
+
+    Returns *list[str]*.
+
+    Scan schemata of data source specified by its id.
```

Comment:
It returned a list of schemas, which exist in the database and can be configured in the data source entity.
Data source managers like Dremio/Drill can work with multiple schemas. 
There, schema name(s) can be injected into scan_request to filter out tables stored in different schemas.